src/base/preferences.h

```diff
@@ -116,6 +116,8 @@ class Preferences : public QObject
     void setHideZeroComboValues(int n);
     bool isStatusbarDisplayed() const;
     void setStatusbarDisplayed(bool displayed);
+    bool isFiltersSidebarDisplayed() const;
+    void setFiltersSidebarDisplayed(bool displayed);
```

Comment:
I don't think you need to add new preferences.

Reply:
We need the preferences in order to persist the user’s set visibility.

Reply:
I think you turn the option into a toggle action instead: "Toggle Filters Sidebar". The visibility setting does persist already.

Reply:
I wouldn't use Preferences to store the state of GUI controls. You can use SettingsStorage directly.

I still have hope to separate the "real" app settings from the saved states of GUI controls/dialogs.

Reply:
In my testing the visibility does not persist on its own. `m_transferListFiltersWidget`'s visibility is reset every time the application is launched.

Reply:
>In my testing the visibility does not persist on its own. 

I suppose you meant the [visible property](https://doc.qt.io/qt-5/qwidget.html#visible-prop), but no I'm not referring to that. Check `m_splitter`, you should control it instead.

Reply:
Updated PR to use `m_splitter` state without needing a separate preference.